datasets/bor_inspecties/dataset.json

```diff
@@ -0,0 +1,302 @@
+          },
+          "closingUserName": {
+            "type": "string",
+            "description": "Gebruiker die de inspectie heeft gefinaliseerd"
+          },
+          "scoreId": {
+            "type": "string",
+            "description": "Het id van het resultaat"
+          },
+          "score": {
+            "type": "string",
+            "enum": [
+              "A+",
+              "A",
+              "B",
+              "C",
+              "D",
+              "NVT"
+            ],
+            "description": "A+, A, B, C, D of NVT"
+          },
+          "scoreCreatedAt": {
+            "type": "string",
+            "description": "Unix timestamp met aanmaakdatum en tijd."
+          },
+          "creatingUserId": {
+            "type": "string",
+            "description": "Id van de inspecteur. Deze id is uniek ongeacht de evironment."
+          },
+          "creatingUserName": {
+            "type": "string",
+            "description": "Volledige naam van de inspecteur."
+          },
+          "photosPhotoUri": {
+            "type": "string",
+            "description": "Link naar de foto"
+          },
+          "xMeting": {
+            "type": "string",
+            "description": "X-coordinaat van meting (RD EPSG:28992)"
+          },
+          "yMeting": {
+            "type": "string",
+            "description": "Y-coordinaat van meting (RD EPSG:28992)"
+          },
+          "geometry": {
+            "type": "string",
+            "profenance": "geometry_coordinates",
+            "description": "Coordinaten van de feature (RD EPSG:28992)",
+            "$ref": "https://geojson.org/schema/Geometry.json"
+          },
+          "inspectionTypeId": {
+            "type": "string",
+            "description": "Id van de beeldmeetlat"
+          },
+          "inspectionTypeUri": {
+            "type": "string",
+            "description": "Link naar de beeldmeetlat info"
+          },
+          "inspectionTypeName": {
+            "type": "string",
+            "description": "Link naar de beeldmeetlat info"
+          },
+          "crowCode": {
+            "type": "string",
+            "description": "CROW unieke code van de beeldmeetlat (RAW code)"
+          },
+          "inspectionTypeOptionId": {
+            "type": "string",
+            "description": "Id van de beeldmeetlat optie"
+          },
+          "inspectionTypeOptionUri": {
+            "type": "string",
+            "description": "Link naar de beeldmeetlat optie"
+          },
+          "inspectionTypeOptionName": {
+            "type": "string",
+            "description": "Naam van de beeldmeetlat optie (bijv. A+)"
+          },
+          "TypeOptionDescription": {
+            "provenance": "inspection_type_option_description",
+            "type": "string",
+            "description": "Omschrijving van de beeldmeetlat optie"
+          },
+          "TypeOptionMeasuring": {
```

Comment:
De veldnamen moeten met een lowercase beginnen. Dus een `t` is wel mogelijk.

Reply:
wordt aangepast.